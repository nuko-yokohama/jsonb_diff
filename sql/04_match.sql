SELECT * FROM jsonb_diff(
  '{"id":1, "name":"foo", "data":"foo", "col":1, "arr":[1,"2",3,4] }',
  '{"id":1, "data":"foo", "col":1, "arr":[1,"2",3,4], "name":"foo" }'
);
