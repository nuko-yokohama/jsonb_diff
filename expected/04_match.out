SELECT * FROM jsonb_diff(
  '{"id":1, "name":"foo", "data":"foo", "col":1, "arr":[1,"2",3,4] }',
  '{"id":1, "data":"foo", "col":1, "arr":[1,"2",3,4], "name":"foo" }'
);
NOTICE:  table "scm_table" does not exist, skipping
 kind | left_path | left_schema | right_path | right_schema 
------+-----------+-------------+------------+--------------
(0 rows)

